app/helpers/goals_helper.rb

```diff
@@ -0,0 +1,11 @@
+module GoalsHelper
+  def time_left_until_expires(goal)
+    time_of_expiration = goal.created_at + 7.days
```

Comment:
Consider [introducing a constant](https://eliotsykes.com/magic-numbers) to represent the `7.days` as this is used in a couple of places now.